doc/docs/modules/ROOT/pages/kafka-ssl.adoc

```diff
@@ -195,5 +195,79 @@ kafka.sasl.mechanism=PLAIN
 
 For more information, please consult the official Confluent documentation at the following links:
 
-* https://docs.confluent.io/2.0.0/kafka/sasl.html#authentication-using-sasl
-* https://docs.confluent.io/2.0.0/kafka/sasl.html#configuring-kafka-clients
+* https://docs.confluent.io/platform/current/kafka/authentication_sasl/index.html
+
+== Authorization with ACL's
+
+When Kafka is configured to use ACL's, you have to add also the following properties on the `neo4j.conf` file:
```

Comment:
To configure use with ACLs, the following configuration properties are required.


(Note:  do not reference neo4j.conf because we're about to introduce dynamic config)

Reply:
Also: does this section of documentation apply only to the plugin?  I think it does.  If so, just make a note.